Gamecube/GamecubeMain.cpp

```diff
@@ -342,10 +342,26 @@ void video_mode_init(GXRModeObj *videomode, u32 *fb1, u32 *fb2, u32 *fb3)
 	xfb[2] = fb3;
 }
 
+bool Autoboot;
+char AutobootROM[1024];
+char AutobootPath[1024];
+
 int main(int argc, char *argv[]) 
 {
 	/* INITIALIZE */
 #ifdef HW_RVL
+	if(argc > 2 && argv[1] != NULL && argv[2] != NULL)
```

Comment:
Why do you need to check against NULL? Is that even possible?

Reply:
from what i understood in this piece of code check if the first argument is a valid path which exists and check if the second argument is a valid file name which exists.
If these are not valid, they will throw NULL and the emulator will start displaying the menu.